client/src/components/_common/SectionContent/SectionContent.layouts.module.css

```diff
@@ -0,0 +1,127 @@
+/* 1 Row */
+
+/* TODO: Duplicate this as '.sidebar-and-content' */
+.one-row {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+}
+
+
+
+
+
+/* 1 Column */
+
+.one-column {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+}
+
+
+
+
+
+/* 2 Columns */
```

Comment:
This is not illustrated in UI Patterns. It is meant specifically for Dashboard. Please await next PR to see this in action (it has been tested in earlier PR #236).